packages/pwa/app/routes.jsx

```diff
@@ -98,9 +100,11 @@ const routes = [
         path: '/account/wishlist',
         component: Wishlist
     },
+    //TODO: Change for testing purposes. Route not matching the PWA routes
+    // are redirected to SFRA site.
     {
         path: '*',
-        component: PageNotFound
+        component: RedirectToSFRA
```

Comment:
Chatted with Adam, it looks like this `RedirectToSFRA` component doesn't provides much value. Developers can achieve the same thing with `component: withRouter(({location) => <Redirect to={location. href} />)`

Reply:
Confirmed `withRouter()` works well, we are not required to create a new `RedirectToSFRA` component.